src/frontend/efiling-frontend/src/components/page/home/Home.stories.js

```diff
@@ -79,7 +79,9 @@ const noAccountExistsComponent = (
   </NoAccountExistsStateData>
 );
 
-export const Loader = () => loaderComponent;
+export const Error = () => loaderComponent;
+
+export const ErrorMobile = () => loaderComponent;
```

Comment:
I purposely removed this story because I dont think we really needed it, but now that it is error state if you think we need it then that's good. Just wondering :)

Reply:
I left it in to make sure the error still looked ok on mobile...not 100% that we need it but figured I would leave/re-add it incase.